lib/assessment.js

```diff
@@ -5,15 +5,19 @@
 // and returns the minimum from that set
 // assume that all the argument are numbers
 var min = function min() {
+  for (i=0; i< arguments.length; i++){
+    return arguments+
+  }
+min(1,4,9);
```

Comment:
Here's the concept. `arguments` is an array-like object that holds all the arguments our function gets. If we want to get the minimum value of those arguments, we can iterate through the array and 'memo-ize' (store between iterations) our minimum value, comparing each element to the minimum value and updating the minimum value accordingly.
i.e.
```javascript
var min = function (){
  var minimum = arguments[0];
  for (var i = 1; i < arguments.length; i++) {
    if (arguments[i] < minimum) {
      minimum = arguments[i];
    }
  }
  return minimum;
};
```